cmd/main.go

```diff
@@ -27,15 +27,23 @@ func main() {
 	viper.SetDefault("PORT", 8080)
 	viper.SetDefault("ADAPTER_URLS", "")
 
+	home, err := os.UserHomeDir()
 	if viper.GetString("USER_DATA_FOLDER") == "" {
-		home, err := os.UserHomeDir()
 		if err != nil {
 			logrus.Fatalf("unable to retrieve the user's home directory: %v", err)
 		}
 		viper.SetDefault("USER_DATA_FOLDER", path.Join(home, ".meshery", "config"))
 	}
 	logrus.Infof("Using '%s' to store user data", viper.GetString("USER_DATA_FOLDER"))
 
+	if viper.GetString("KUBECONFIG_FOLDER") == "" {
+		if err != nil {
+			logrus.Fatalf("unable to retrieve the user's home directory: %v", err)
```

Comment:
In a future PR, we'll want to account for in-cluster deployments by looking for the locally mounted kubeconfig (when deployed inside a Kubernetes cluster).